cmd/ocitools/validate.go

```diff
@@ -340,6 +336,14 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 		}
 	}
 
+	if (len(spec.Linux.UIDMappings) > 0 || len(spec.Linux.GIDMappings) > 0) && !userExists {
+		msgs = append(msgs, "UID/GID mappings require User namespace exists")
```

Comment:
On 08/24/2016 12:50 AM, W. Trevor King wrote:
> The phrasing used by existing similar warnings is:
> 
>     {setting} requires a new {type} namespace to be specified as well
> 
I think container joins into an existing user namespace also can work.
Doesn't it?